### PR TITLE
Added discard button to image widget

### PR DESCRIPTION
**Pull request by rristovic**
-Product note: Added cancel option on image question to remove taken image.
Jira task: https://dimagi-dev.atlassian.net/browse/PF-310

When there is selected there is remove button

with remove button

When there is no selected image, remove button is hidden.
without remove button

### CODE REVIEW

File: app/res/values-pt/strings.xml [hunk 1]
```diff
@@ -38,7 +38,8 @@
     <string name="delete_file">Apagar Selecionados</string>
     <string name="delete_no">Não Apagar</string>
     <string name="delete_yes">Apagar Formulários</string>
-    <string name="discard_answer">Remove response</string>
+    <string name="discard_answer">Remover resposta</string>
```
Comment (shubham1g5) [inline app/res/values-pt/strings.xml]: Thanks for the correction here.

**Comment by shubham1g5** [comment]
Hey, lets only merge this after we fix this - https://dimagi-dev.atlassian.net/browse/MOB-72 .

File: app/res/values-pt/strings.xml (hunk 1 above)
Comment (rristovic) [reply app/res/values-pt/strings.xml]: You're welcome ;)

**Comment by shubham1g5** [comment]
Merging this now since the [MOB-72](https://dimagi-dev.atlassian.net/browse/MOB-72) is now fixed.

**Pull request merged**